.yarn/patches/eth-trezor-keyring-npm-0.10.0-1bf9aa3f4d.patch

```diff
@@ -0,0 +1,87 @@
+diff --git a/index.js b/index.js
```

Comment:
Ignore this patch - a discussion regarding changes is occurring in the related repo.